### PR TITLE
Update to Arm Debugger and tested with latest CMSIS Csolution extension

### PR DESCRIPTION
**Pull request by jreineckearm**
This PR prepares the example for use with Arm Debugger.
Tested with latest CMSIS Csolution extension.

### CODE REVIEW

**Comment by jreineckearm** [comment]
@KeilChris , @RobertRostohar , please wait with merge until we have switched Arm Keil Studio Pack to ship with the Arm Debugger extension.

**Comment by jreineckearm** [comment]
More updates necessary next week.

File: .vscode/tasks.json
Comment (mcgordonite) [inline .vscode/tasks.json:1]: Please also remove the CMSIS Build task as this is no longer necessary: the Csolution extension's build button doesn't depend on this task being present. Also, some of the commands referenced have been removed.

Comment (jreineckearm) [reply .vscode/tasks.json:1]: Agreed, idea is to replace the `.vscode` files and `vcpkg-configuration.json` with the defaults from the CMSIS Csolution extension's `Initialize CMSIS project` functionality.

**Comment by jreineckearm** [comment]
@KeilChris , @mcgordonite , please review.

File: .vscode/tasks.json [hunk 1]
```diff
@@ -1,27 +1,15 @@
...
+    "tasks": [
+        {
+            "type": "arm-debugger.flash",
+            "serialNumber": "${command:device-manager.getSerialNumber}",
+            "program": "${command:arm-debugger.getApplicationFile}",
+            "cmsisPack": "${command:device-manager.getDevicePack}",
+            "deviceName": "${command:cmsis-csolution.getDeviceName}",
+            "processorName": "${command:cmsis-csolution.getProcessorName}",
+            "problemMatcher": [],
+            "label": "arm-debugger.flash: Flash Device"
```
Comment (mcgordonite) [inline .vscode/tasks.json]:
```suggestion
            "label": "Flash Device"
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "cmsis-csolution.actions": [
+        {
+            "path": "hello.csolution.yaml",
+            "target": "FRDM-K32L3A6",
+            "config": {
+                "run": "arm-debugger.flash: Flash Device",
```
Comment (mcgordonite) [inline .vscode/settings.json]:
```suggestion
                "run": "Flash Device",
```

**Review by mcgordonite — APPROVED** [review]

**Review by KeilChris — APPROVED** [review]
LGTM

**Pull request merged**